pl/mozorg/contribute/index.lang

```diff
@@ -499,104 +499,104 @@ Open Web science research
 
 
 ;Community Events
-Community Events
+Wydarzenia społeczności
 
 
 ;Find events
-Find events
+Znajdź wydarzenia
 
 
 ;By city, country, or event:
-By city, country, or event:
+Według miasta, kraju lub wydarzenia:
 
 
 ;By date:
-By date:
+Według daty:
 
 
 ;month
-month
+miesiąc
 
 
 ;year
-year
+rok
 
 
 ;Date
-Date
+Data
 
 
 ;Event
-Event
+Wydarzenie
 
 
 ;Location
-Location
+Miejsce
 
 
 ;City
-City
+Miasto
 
 
 ;Events
-Events
+Wydarzenia
 
 
 # Used as title of the thank you page displayed after users send the form
 ;Thank you!
-Thank you!
+Dziękujemy!
 
 
 ;Thanks!
-Thanks!
+Podziękowania!
 
 
 ;We’re thrilled to see that you’re interested in coding and can’t wait to get you started.
-We’re thrilled to see that you’re interested in coding and can’t wait to get you started.
+Jesteśmy zachwyceni widząc, że jesteś zainteresowany/a programowaniem i nie możemy doczekać się, aż zaczniesz.
```

Comment:
„Widząc” można wywalić.
Możemy uniknąć płci na przykład pisząc „wyrażasz zainteresowanie”.
”się doczekać”.

To samo dotyczy kilku następnych ciągów.